include/nudb/error.hpp

```diff
@@ -81,6 +81,12 @@ namespace errc = boost::system::errc;
 /// Database error codes.
 enum class error
 {
+    /** Operation succeeded
+
+        Default error_code value.
+     */
+    success
```

Comment:
LOL... or just use `key_not_found = 1` :)

Reply:
I like @movitto's suggestion as it follows boost.

Reply:
We can do that, but make sure to write `success = 0` so that it is explicit

Reply:
@vinniefalco you got it!